### ISSUE TITLE
Reading ad macro file fails when webppl is used as dependence

### ISSUE THREAD

**Issue by stuhlmueller** [probmods/webppl#382]
In `src/main.js`, we read the ad macro file as follows:

    var adMacroModule = fs.readFileSync(__dirname + '/../node_modules/ad.js/macros/index.js', 'utf8');

This can be problematic: newer versions of npm flatten the dependency structure in `node_modules` and eliminate duplicate dependencies, so if a project requires `webppl` as a dependence, `ad.js` may not be found in the location given here.

To reproduce the error, run:

~~~~
mkdir my-new-project
cd my-new-project/
npm init -y
npm install --save webppl
echo "flip(.5)" > test.wppl
./node_modules/webppl/webppl test.wppl
~~~~

This results in the following error message:

~~~~
Processing test.wppl
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                                                  ^

Error: ENOENT: no such file or directory, open '/private/tmp/my-new-project/node_modules/webppl/src/../node_modules/ad.js/macros/index.js'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:431:33)
    at headerPackage (/private/tmp/my-new-project/node_modules/webppl/src/main.js:74:26)
    at _parsePackageCode (/private/tmp/my-new-project/node_modules/webppl/src/main.js:111:24)
    at Object.timeif (/private/tmp/my-new-project/node_modules/webppl/src/util.js:222:37)
    at Object.parsePackageCode (/private/tmp/my-new-project/node_modules/webppl/src/main.js:123:15)
    at run (/private/tmp/my-new-project/node_modules/webppl/webppl:41:23)
    at main (/private/tmp/my-new-project/node_modules/webppl/webppl:134:3)
    at Object.<anonymous> (/private/tmp/my-new-project/node_modules/webppl/webppl:137:1)
~~~~

**Comment by null-a** [comment]
Some background: The reason the macros are loaded this way is so that browserify/brfs does the right thing with them. More recent versions of brfs know how to (statically) handle `require.resolve` which is exactly what we need here. The problem is that we're stuck on an old version of brfs because the current version doesn't bundle sweet.js correctly.

Ideally, we'd tackle #270, but that might be a long and painful piece of work. So instead, how about we just copy the ad macros into WebPPL's `headerMacros.sjs` until we can do something better?

**Comment by null-a** [comment]
Closed by #383.
